webpack.config.js

```diff
@@ -1,86 +1,76 @@
-const CopyWebpackPlugin = require('copy-webpack-plugin');
 const path = require('path');
-const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const webpack = require('webpack');
+
+const TerserPlugin = require('terser-webpack-plugin');
```

Comment:
Can we skip Terser here and rely on webpack's `minimize` flag?

Reply:
@cwillisf According to the webpack documentation for minimize:
> Tell webpack to minimize the bundle using the [TerserPlugin](https://webpack.js.org/plugins/terser-webpack-plugin/) or the plugin(s) specified in [optimization.minimizer](https://webpack.js.org/configuration/optimization/#optimizationminimizer).

In `scratch-webpack-configuration` we set `minimize` to true if the environment is production. So i guess we can remove the `minimizer` from the configuration here ans depend on what is set in the template.

Reply:
That sounds good to me